src/gt4py/next/otf/compilation/build_systems/cmake_lists.py

```diff
@@ -45,12 +45,6 @@ class CMakeListsGenerator(eve.codegen.TemplatedGenerator):
         project({{project_name}})
 
         # Languages
-        if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
```

Comment:
Maybe instead of reverting the changes, we could generate a script that contains the cmake invocation command (maybe including the `env` when the command was executed). This leaves the cleaner version as implemented here, but allows to re-run from scratch.

Reply:
I came to this idea, because initially I wanted to propose to write down: our strategy is to put all the information in the cmakelists.txt such that it can be re-run, then I realized that this would also mean putting the compilers etc into the file, which might or might not be what the person wants that tries to re-configure.